db/migrate/20150316135303_create_course_enrol_requests.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCourseEnrolRequests < ActiveRecord::Migration
+  def change
+    create_table :course_enrol_requests do |t|
```

Comment:
Though I believe enrol can mean to register someone for a class, I think we can call it registration. I am planning to name the other two ways as mass invitation and secret code invitation. So registration looks to be a good fit as it shows the direction more clearly.